addons/website_slides/tests/test_slide_question.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import Command
+
+from odoo.addons.website_slides.tests import common as slides_common
+from odoo.tests import tagged
+from odoo.tests.common import users
+
+@tagged('slide_question')
+class TestSlideQuestionManagement(slides_common.SlidesCase):
+
+    @users('user_officer')
+    def test_compute_answers_validation_error(self):
+        channel = self.env['slide.channel'].create({
+            'name': 'Test compute answers channel',
+            'slide_ids': [Command.create({
+                'name': "Test compute answers validation error slide",
+                'slide_category': 'quiz',
+                'question_ids': [Command.create({
+                    'question': 'Will test compute answers validation error pass?',
+                    'answer_ids': [
+                        Command.create({
+                            'text_value': 'An incorrect answer',
+                        }),
+                        Command.create({
+                            'is_correct': True,
+                            'text_value': 'A correct answer',
+                        })
+                    ]
+                })]
+            })]
+        })
+
```

Comment:
to have 100% coverage on the condition, can we try
```
[False, False], [True, True]
```

(can be done in a loop to remove redundacy)